archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -0,0 +1,87 @@
+import tactic
+import algebra.big_operators
+import data.real.basic
+
+lemma range_insert (n : ℕ) :
```

Comment:
We already have this in `mathlib`, see `finset.range_succ` and `prod_range_succ`.

Reply:
deleted range_insert and replaced its uses with calls to existing lemmas